VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -90,7 +97,10 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
         @DrawableRes
         final int backgroundRes;
         final IconDrawable iconDrawable;
+        final int standardMargin = context.getResources().getDimensionPixelOffset(R.dimen.discussion_responses_standard_margin);
         if (position == 0) {
+            holder.discussionCommentRow.setPadding(0, standardMargin, 0, 0);
+            DiscussionTextUtils.setEndorsedState(holder.responseAnswerTextView, thread, response);
```

Comment:
This needs to be hidden in the other rows. It might also make sense to create a separate layout for this header view, which would remove much of the customizing logic from here, and remove the useless extra text view in standard rows. It's up to you though.